src/lib.rs

```diff
@@ -69,7 +77,7 @@ impl std::error::Error for VMError {}
 pub struct Handle<'a> {
     handle: *mut WrenHandle,
     wvm: *mut WrenVM,
-    vm: marker::PhantomData<&'a VM>
+    vm: marker::PhantomData<&'a VM>,
```

Comment:
There seem to be a lot of formatting changes that I'm at odds with, so I'mma summarize and stop harping on them:

- Try to keep code compact unless it affect readability (don't split lines unless you need to for keeping the code coherent)
- No trailing commas unless the data will change in the foreseeable future.

Reply:
oh woops this is all rustfmt, so I should bother them. Anyways, don't run rustfmt on the code with this change, make that a separate commit. removing my earlier comments. Turns out I don't write standard-format Rust XD